.github/workflows/cron-weeklySummary.yml

```
@@ -9,6 +9,9 @@ on:
     - cron: "0 8 * * 1"
 jobs:
   cron-weeklySummary:
+    permissions:
+      contents: read    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)